Tools/CISupport/ews-build/steps.py

```diff
@@ -1770,7 +1770,8 @@ def start(self):
 
         if not reviewers:
             # Change has not been reviewed in bug tracker. This is acceptable, since the ChangeLog might have 'Reviewed by' in it.
-            self.descriptionDone = 'Validated committer'
+            self._addToLog('stdio', f'Reviewer not found. Commit message  will be checked for reviewer name in later steps\n')
+            self.descriptionDone = 'Validated committer, Reviewer not found'
```

Comment:
Nit: since commiter is lowercase, let's make reviewer lowercase too.

Reply:
Will do.